selfie-lib/src/commonMain/kotlin/com/diffplug/selfie/SourceFile.kt

```diff
@@ -73,20 +74,45 @@ class SourceFile(filename: String, content: String) {
      * `toBe_TODO()`.
      */
     fun <T : Any> parseLiteral(literalFormat: LiteralFormat<T>): T {
-      // this won't work, because we need to find the `.toBe` and parens
-      TODO("return literalFormat.parse(slice.toString())")
+      return literalFormat.parse(
+          slice.subSequence(valueStart, slice.length - 1).toString(), language)
     }
   }
   fun parseToBe_TODO(lineOneIndexed: Int): ToBeLiteral {
+    return parseToBeLike(".toBe_TODO(", lineOneIndexed)
+  }
+  fun parseToBe(lineOneIndexed: Int): ToBeLiteral {
+    return parseToBeLike(".toBe(", lineOneIndexed)
+  }
+  private fun parseToBeLike(prefix: String, lineOneIndexed: Int): ToBeLiteral {
     val lineContent = contentSlice.unixLine(lineOneIndexed)
-    val idx = lineContent.indexOf(".toBe_TODO()")
+    val idx = lineContent.indexOf(prefix)
     if (idx == -1) {
       throw AssertionError(
-          "Expected to find `.toBe_TODO()` on line $lineOneIndexed, but there was only `${lineContent}`")
+          "Expected to find `$prefix)` on line $lineOneIndexed, but there was only `${lineContent}`")
     }
-    return ToBeLiteral(lineContent.subSequence(idx, idx + ".toBe_TODO()".length))
-  }
-  fun parseToBe(lineOneIndexed: Int): ToBeLiteral {
-    TODO("More complicated because we have to actually parse the literal")
+    var opened = 0
+    val startIndex = idx + prefix.length
+    var endIndex = -1
+    // TODO: do we need to detect paired parenthesis ( ( ) )?
```

Comment:
We do, but we can add that in a future PR. It is complicated because we might have:

```java
.toBe("""
(((
(((
""")
// or
.toBe(
  "(((\n" +
   "(((")
```